src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,48 @@
+import React = require('react');
+
+interface CheckBoxProps {
+    text?: string;
+    value?: boolean;
+    boxIcon?: string;
+    tickIcon?: string;
+    onChange?: (value: boolean ) => any;
+
+}
+
+export class CheckBox extends React.Component<CheckBoxProps, {}> {
```

Comment:
@tyv we discussed it and agree with your position. I'll refactor. Thanks for the input!